ci-operator/step-registry/ipi/conf/aws/windows-machineset/ipi-conf-aws-windows-machineset-ref.yaml

```diff
@@ -16,6 +16,8 @@ ref:
       default: "m5a.large"
     - name: WINDOWS_NODE_REPLICAS
       default: "1"
+    - name: WINDOWS_OS_ID
+      default: "Windows_Server-2019-English-Full-Containers"
```

Comment:
To speed up the PR merge, we can take the changes in `ci-operator/step-registry/ipi/conf/aws/windows-machineset` out and use a follow up PR for it.

Reply:
Done, thanks for the suggestion. Submitted a new PR with the other change at https://github.com/openshift/release/pull/37125